src/quacc/schemas/ase.py

```diff
@@ -255,12 +255,18 @@ def summarize_vib_and_thermo(
     vib_task_doc = _summarize_vib_run(
         vib, charge_and_multiplicity=charge_and_multiplicity
     )
-    thermo_task_doc = _summarize_ideal_gas_thermo(
-        igt,
-        temperature=temperature,
-        pressure=pressure,
-        charge_and_multiplicity=charge_and_multiplicity,
-    )
+
+    if isinstance(thermo_analysis, HarmonicThermo):
+        thermo_task_doc = _summarize_harmonic_thermo(
+            ht=thermo_analysis, temperature=temperature, pressure=pressure
+        )
```

Comment:
This line will get covered when you make the minimal VASP harmonic thermo test and/or when you add a brief test for `summarize_vib_and_thermo` directly as commented below.